applications/PfemFluidDynamicsApplication/custom_strategies/strategies/two_step_v_p_strategy.h

```diff
@@ -269,8 +271,8 @@ namespace Kratos
         if (it == maxNonLinearIterations - 1 || ((continuityConverged && momentumConverged) && it > 2))
         {
 
-          //double tensilStressSign = 1.0;
-          // ComputeErrorL2Norm(tensilStressSign);
+          // double tensilStressSign = 1.0;
+          //  ComputeErrorL2Norm(tensilStressSign);
```

Comment:
unused

Reply:
thanks as usual for the comments @AlejandroCornejo 
I have made the changes requested. I'll make the utility for constitutive law in a separate PR, if it's ok for you.